tests/compute/vmidefaults.go

```diff
@@ -45,31 +44,23 @@ import (
 )
 
 var _ = Describe(SIG("VMIDefaults", func() {
-	var virtClient kubecli.KubevirtClient
-
-	var vmi *v1.VirtualMachineInstance
-
-	BeforeEach(func() {
-		virtClient = kubevirt.Client()
-	})
```

Comment:
Not sure how much of an improvement this change is

Reply:
Matter of taste